systest/online-restore/docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
       source: ../../tlstest/mtls_internal/tls/alpha1
       target: /dgraph-tls
       read_only: true
-    command: /gobin/dgraph alpha --my=alpha1:7080 --zero=zero1:5080
+    command: dgraph alpha --my=alpha1:7080 --zero=zero1:5080
```

Comment:
Can the above /gobin bind point be removed?

Reply:
fixed, thanks